chef-config/lib/chef-config/windows.rb

```diff
@@ -15,14 +15,10 @@
 # limitations under the License.
 #
 
-module ChefConfig
+require "chef-utils"
 
+module ChefConfig
   def self.windows?
```

Comment:
do we want to start warning to not use this method anymore and instead use the utils method?